packages/pages/src/vite-plugin/build/closeBundle/bundleValidator.test.ts

```diff
@@ -0,0 +1,96 @@
+import { glob } from "glob";
+import fs, { Stats } from "fs";
+import { validateBundles } from "./bundleValidator";
+
+describe("bundleValidator", () => {
+  it("throws an error when a single file is over the max filesize limit", async () => {
+    const stats = getStats(1);
+
+    jest.spyOn(glob, "sync").mockImplementation(() => ["file1.js", "file2.js"]);
```

Comment:
make sure to restoreAllMocks
https://jestjs.io/docs/jest-object#jestspyonobject-methodname